.github/workflows/CI-workflow.yml

```diff
@@ -145,20 +145,20 @@ jobs:
         if: env.imagePresent == 'true'
         run: |
           cd ..
-          docker run -p 9200:9200 -d -p 9600:9600 -e "discovery.type=single-node" opensearch-ml:test
+          docker run -p 9200:9200 -d -p 9600:9600 -e "discovery.type=single-node" -e OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! opensearch-ml:test
```

Comment:
Why not just use the previous default username and password admin:admin for BWC?

Reply:
admin is too weak, need have a strong one.

Reply:
Let's avoid using hard coded password in code

Reply:
done by replaced with generated one

Reply:
docker run success in my fork repo https://github.com/Hailong-am/ml-commons/actions/runs/7497254147/job/20410737212